at.nis.one_click_backup.yml

```diff
@@ -0,0 +1,30 @@
+{
+    "app-id": "at.nis.one_click_backup",
```

Comment:
the `nis.at` domain is for sale.

the requirement in the checklist isn't optional.

Reply:
Okay, but what should I use instead? I'm not really able to afford buying a domain just for this (very simple) app.

Reply:
The documentation indicate you can use something associated to the code hosting domain.

Reply:
So like `com.gitlab.dev-nis.one_click_backup`? Does it have to use the exact username and repository-name?

Reply:
you need to escape it only in the domain component https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic ie. the first two here.

Reply:
Thank's for that hint! I will rename my files and push another commit

Reply:
Renamed files and app-id with commit [32eb2a6278c235c5ad180ad40d7d981d2a74c666](https://github.com/flathub/flathub/pull/4864/commits/32eb2a6278c235c5ad180ad40d7d981d2a74c666)